designs/administrator/olm/operand-details-grouping/index.md

```diff
@@ -0,0 +1,35 @@
+---
+parent: Administrator
+version: 4.6
+---
+
+# Improve grouping and editing on Operand Details view
+
+When creating an operand, fairly complex nesting and hierarchy of fields is used to set values on the operand. Once created however, the Details tab for that operand does not show this hierarchy and all fields are shown at the same flat level, with somewhat of a mix of spec and status descriptors, with some hierarchical section headers missing as well. This can make this view challenging to interpret.
+
+This design describes surfacing the common spec descriptors first, the status descriptors next, then the rest of the spec descriptors, with a divider separating them. Groups of spec descriptors will now have their grouping reflected using a box for the first level, with the common `DetailsItem` popover to surface the full hierarchy. An option to edit these groups using their existing form view will also be available.
+
+## Separating Spec and Status descriptors
+
+![Dividing groups](img/8-1-divider.png)
```

Comment:
curious if you are using the PF divider component here
https://www.patternfly.org/v4/documentation/core/components/divider

Reply:
I believe so? @TheRealJon

Reply:
We aren't using that component actually. The those separators on the details page layout pre-date PF adoption.